.stylelintrc

```diff
@@ -8,7 +8,7 @@
     "scss/at-rule-no-unknown": true,
     "at-rule-empty-line-before": null,
     "no-empty-source": null,
-    "unit-whitelist": ["em", "rem", "%", "px", "deg", "vh", "vw", "ms", "s", "fr"],
+    "unit-allowed-list": ["em", "rem", "%", "px", "deg", "vh", "vw", "ms", "s", "fr"],
```

Comment:
Stylelint was giving a deprecation warning about this renamed config value.

Reply:
These were added in: https://github.com/stylelint/stylelint/releases/tag/13.7.0 So I think we need to also update package.json to `"stylelint": "^13.7.0"`